tests/test_masks.py

```diff
@@ -0,0 +1,165 @@
+    ("1234-5678", "Номер счета должен содержать только цифры"),
+    (None, "Номер счета должен быть строкой"),
+    (False, "Номер счета должен быть строкой"),
+    ([], "Номер счета должен быть строкой"),
+    ({}, "Номер счета должен быть строкой"),
+    (5.67, "Номер счета должен быть строкой"),
+])
+def test_get_mask_account_invalid(invalid_account, expected_error):
+    with pytest.raises((ValueError, TypeError)) as exc_info:
+        get_mask_account(invalid_account)
+    assert str(exc_info.value) == expected_error
+
+
+# Дополнительные тесты для полного покрытия
+def test_get_mask_card_number_immutability():
+    original = "1234567890123456"
+    masked = get_mask_card_number(original)
+    assert original == "1234567890123456"
+    assert original is not masked
+
+
+def test_get_mask_account_immutability():
+    original = "12345678"
+    masked = get_mask_account(original)
+    assert original == "12345678"
+    assert original is not masked
+
+
+def test_get_mask_card_number_return_type():
+    result = get_mask_card_number("1234567890123456")
+    assert isinstance(result, str)
+
+
+def test_get_mask_account_return_type():
+    result = get_mask_account("12345678")
+    assert isinstance(result, str)
+
+
+def test_get_mask_card_number_exact_min_length():
+    result = get_mask_card_number("123456")
+    assert result == "1234 56** **** 3456"
+
+
+def test_get_mask_account_exact_min_length():
+    result = get_mask_account("1234")
+    assert result == "**1234"
+
+
+def test_get_mask_card_number_with_whitespace():
+    with pytest.raises(ValueError):
+        get_mask_card_number(" 1234567890123456 ")
+
+
+def test_get_mask_account_with_whitespace():
+    with pytest.raises(ValueError):
+        get_mask_account(" 12345678 ")
+
+
+def test_get_mask_card_number_special_chars():
+    with pytest.raises(ValueError):
+        get_mask_card_number("1234-5678-9012-3456")
+
+
+def test_get_mask_account_special_chars():
+    with pytest.raises(ValueError):
+        get_mask_account("1234-5678")
+
+
+def test_get_mask_card_number_with_letters():
+    with pytest.raises(ValueError):
+        get_mask_card_number("1234abcd56789012")
+
+
+def test_get_mask_account_with_letters():
+    with pytest.raises(ValueError):
+        get_mask_account("abcd5678")
+
+
+def test_get_mask_account_input_unchanged():
+    original = "12345678"
+    original_copy = original[:]
+    _ = get_mask_account(original)
+    assert original == original_copy
+
+
+def test_get_mask_card_number_formatting():
+    result = get_mask_card_number("1234567890123456")
+    parts = result.split()
+    assert len(parts) == 4
+    assert parts[0] == "1234"
+    assert parts[1] == "56**"
+    assert parts[2] == "****"
+    assert parts[3] == "3456"
+
+
+@pytest.mark.parametrize("invalid_input", [
+    b"1234567890123456",  # bytes
+    bytearray(b"1234567890123456"),
+    ("1" * 6,),  # tuple
+    {"card": "1234567890123456"},  # dict
+])
```

Comment:
Это правильнее задавать через ids (после запуска тестов с флагом -v увидишь описание), вместо комментариев
```suggestion
], ids=['bytes', 'byte array', 'tuple', 'dict'])
```